code/modules/mob/living/carbon/xenomorph/castes/hunter/abilities_hunter.dm

```diff
@@ -285,14 +339,16 @@
 	var/pounce_range = HUNTER_POUNCE_RANGE
 
 /datum/action/ability/activable/xeno/pounce/on_cooldown_finish()
-	owner.balloon_alert(owner, "Pounce ready")
-	owner.playsound_local(owner, 'sound/effects/alien/new_larva.ogg', 25, 0, 1)
+	owner.balloon_alert(owner, "[src] ready")
+	owner.playsound_local(owner, 'sound/effects/alien/new_larva.ogg', 25, 0, TRUE)
 	return ..()
 
 /datum/action/ability/activable/xeno/pounce/can_use_ability(atom/A, silent = FALSE, override_flags)
 	. = ..()
 	if(!.)
 		return FALSE
+	if(owner.status_flags & INCORPOREAL)
+		return FALSE
```

Comment:
As above you can just call it first to avoid unneeded proc calls

Reply:
this isnt even my code, wha.